website/static/js/registrationUtils.js

```diff
@@ -246,27 +246,27 @@ var Draft = function(params, metaSchema) {
     self.updated = new Date(params.updated);
     self.completion = ko.computed(function() {
         var total = 0;
-        var complete = 0;        
+        var complete = 0;
         if (self.schemaData) {
             var schema = self.schema();
             $.each(schema.pages, function(i, page) {
                 $.each(page.questions, function(qid, question) {
-                    var q = self.schemaData[qid];                    
+                    var q = self.schemaData[qid];
                     if(q && !$osf.isBlank(q.value)) {
                         complete++;
                     }
                     total++;
                 });
             });
-           return Math.ceil(100 * (complete / total));
+			return Math.ceil(100 * (complete / total));
```

Comment:
@barbour-em It looks like your editor is doing some reindentation/formatting of code. 

This indent for example, is wrong. 

Additionally (although non-critical), avoid reformatting comments unless they are obviously wrong.